components/efuse/src/esp_efuse_utility.c

```diff
@@ -473,6 +500,8 @@ void esp_efuse_utility_write_efuses_to_flash(void)
         ESP_EARLY_LOGE(TAG, "secure_version can not be written to flash. err = 0x%x", err);
         abort();
     }
+
+    ESP_EARLY_LOGW(TAG, "Write complete");
```

Comment:
I think it will overload the logs. If we have an error then it will be catched by the above conditions. I suggest removing it.

Reply:
Writing fuses is very, very, rare, (of course, by design...) so I think there is small risk of overloading.

I've hit conditions where we got stuck (I forget why). I would have appreciated this log.